packages/file-asset-apis/src/lib/slice.ts

```diff
@@ -1,6 +1,37 @@
-import { DataEntity, isString } from '@terascope/job-components';
+import { isString } from '@terascope/job-components';
 import path from 'path';
-import { SliceConfig, SlicedFileResults, Offsets } from './interfaces';
+import { SliceConfig, SlicedFileResults, Offsets } from '../interfaces';
+
+/**
+ *  Used to slice up a file based on the configuration provided, this is a
+ * higher level API, prefer this over getOffsets as that is a lower level API
+ *
+ * The returned results can be used directly with any "read" method of a reader API
+ *
+ * @example
+ *  const slice = { path: 'some/path', size: 1000 };
+    const config = {
+        file_per_slice: false,
+        line_delimiter: '\n',
+        size: 300,
+        format: Format.ldjson
+    };
+
+    const results = segmentFile(slice, config);
+
+    results === [
```

Comment:
This looks odd expressed like this, try just this:

```suggestion
    // outputs => [
```